scipy/sparse/bsr.py

```diff
@@ -407,14 +407,14 @@ def tobsr(self,blocksize=None,copy=False):
         else:
             return self
 
-    def tocsr(self):
-        return self.tocoo(copy=False).tocsr()
+    def tocsr(self, copy=False):
+        return self.tocoo(copy=copy).tocsr()
```

Comment:
I actually was tempted to change this to:

```
if copy:
    # One of the following depending on which is more efficient:
    # return self.tocoo(copy=False).tocsr(copy=True)
    return self.tocoo(copy=True).tocsr(copy=False)
else:
    return self.tocoo(copy=False).tocsr(copy=False)
```

Also, I am not sure I see a reason why `copy=False` should be the default. 
I'll be very surprised if it effects the correctness of the copying. (Does it?)